tests/test_suites/ShortSystemTests/test_read_cache_ec_data_loss.sh

```diff
@@ -0,0 +1,61 @@
+timeout_set 15 minutes
+
+USE_RAMDISK=YES \
+	CHUNKSERVERS=5 \
+    MOUNT_EXTRA_CONFIG="sfscachemode=NEVER`
+			`|readcachemaxsizepercentage=1`
+			`|sfsioretries=1`
+			`|maxreadaheadrequests=0`
+			`|cacheexpirationtime=10000" \
+	MASTER_CUSTOM_GOALS="10 ec_4_1: \$ec(4,1)" \
+	setup_local_empty_saunafs info
+
+dir="${info[mount0]}/dir"
+mkdir "$dir"
+saunafs setgoal ec_4_1 "$dir"
+FILE_SIZE=64M file-generate "$dir/file"
+
+saunafs fileinfo "$dir/file"
+
+# Delete data in part 2 of the chunk
+csid=$(find_first_chunkserver_with_chunks_matching \
+	"chunk_ec2_2_of_4*${chunk_metadata_extension}")
+hdd=$(get_metadata_path "${info[chunkserver${csid}_hdd]}")
+chunk=$(find "$hdd" -name "chunk_ec2_2_of_4_*${chunk_metadata_extension}")
+rm "$chunk"
+
+# Delete data in part 3 of the chunk
+csid=$(find_first_chunkserver_with_chunks_matching \
+	"chunk_ec2_3_of_4*${chunk_metadata_extension}")
+hdd=$(get_metadata_path "${info[chunkserver${csid}_hdd]}")
+chunk=$(find "$hdd" -name "chunk_ec2_3_of_4_*${chunk_metadata_extension}")
+rm "$chunk"
+
+# Delete data in part 4 of the chunk
```

Comment:
Yes, the chunk is lost but for the tests purposes it is just necessary to be able to read at least the first part. The goal of the test was to proof that after the damage of some file in many of its chunks, and then tried to read this files many times as our client retries, this could fill the read cache, and we are checking that after these retries that fill the cache, then we wait and we are able to make at least one read operation as it is expected the cache to be cleaned correctly and avoid any read cache memory no availability errors.